src/components/TodoContainer.js

```diff
@@ -2,38 +2,94 @@ import React from 'react';
 import style from "./TodoContainer.module.css";
```

Comment:
This component is doing a lot. There is a lot of logic that doesn't directly pertain to the todo container. This works fine and it is well structured but it can be cleaned up by having API functionality imported.